libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -31,17 +31,15 @@ func getIPVersion(config *networkConfiguration) ipVersion {
 	return ipVer
 }
 
-func setupBridgeNetFiltering(config *networkConfiguration, i *bridgeInterface) error {
-	err := checkBridgeNetFiltering(config, i)
-	if err != nil {
+func setupBridgeNetFiltering(config *networkConfiguration, _ *bridgeInterface) error {
+	if err := checkBridgeNetFiltering(config); err != nil {
 		if ptherr, ok := err.(*os.PathError); ok {
 			if errno, ok := ptherr.Err.(syscall.Errno); ok && errno == syscall.ENOENT {
 				if isRunningInContainer() {
 					log.G(context.TODO()).Warnf("running inside docker container, ignoring missing kernel params: %v", err)
-					err = nil
-				} else {
-					err = errors.New("please ensure that br_netfilter kernel module is loaded")
+					return nil
 				}
+				err = errors.New("please ensure that br_netfilter kernel module is loaded")
 			}
 		}
 		if err != nil {
```

Comment:
Now that the preceding code no longer nils out `err`, this condition will always be true.

Reply:
Will probably pick that one up as well in "the sequel" 👍